src/assets/css/main.css

```diff
@@ -1234,7 +1238,13 @@ ul {
     display: -ms-flexbox;
     display: flex;
     padding: 1rem;
-    background-color: #334a51
+    background-color: #334a51;
+    transform: translateY(0);
+    transition: transform 0.35s ease-in-out;
```

Comment:
Consider slowing this down a little bit. Just personal preference, but I feel like it exits kind of quickly.

Reply:
I'll up it to .5s to see if that helps.